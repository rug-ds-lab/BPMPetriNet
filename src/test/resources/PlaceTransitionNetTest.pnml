<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<pnml>
    <net id="t1" type="http://www.pnml.org/version-2009/grammar/petrinet">
        <arc source="yy0" target="xx1" id="yy0-x1"/>
        <arc source="xx0" target="yy0" id="xx0-yy0">
            <inscription>
                <text>2</text>
            </inscription>
        </arc>
        <page id="page0">
            <toolspecific tool="nl.rug.ds.bpm.ptnet" version="1.0">
                <process>
                    <groups/>
                    <roles/>
                    <variables/>
                </process>
            </toolspecific>
            <place id="place01"/>
        </page>
        <place id="xx1">
            <initialMarking>
                <text>1</text>
            </initialMarking>
        </place>
        <place id="xx0">
            <name>
                <text>test_xx0</text>
            </name>
        </place>
        <toolspecific tool="nl.rug.ds.bpm.ptnet" version="1.0">
            <process>
                <groups/>
                <roles/>
                <variables>
                    <variable name="x" type="int" value="0"/>
                </variables>
            </process>
        </toolspecific>
        <transition id="yy0">
            <name>
                <text>ytest</text>
            </name>
            <toolspecific tool="nl.rug.ds.bpm.ptnet" version="1.0">
                <task isTau="false">
                    <guard>i&gt;=0</guard>
                    <script type="js">i++;</script>
                </task>
            </toolspecific>
        </transition>
    </net>
</pnml>
