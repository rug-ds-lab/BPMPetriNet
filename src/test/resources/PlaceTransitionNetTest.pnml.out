<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<pnml>
    <net id="t1" type="http://www.pnml.org/version-2009/grammar/petrinet">
        <arc source="y0" target="x1" id="y0-x1"/>
        <arc source="xx0" target="yy0" id="xx0-yy0">
            <inscription>
                <text>2</text>
            </inscription>
        </arc>
        <arc source="x0" target="y0" id="x0-y0">
            <inscription>
                <text>2</text>
            </inscription>
        </arc>
        <arc source="yy0" target="xx1" id="yy0-x1"/>
        <page id="page0">
            <toolspecific tool="nl.rug.ds.bpm.ptnet" version="1.0">
                <process>
                    <groups/>
                    <roles/>
                    <variables/>
                </process>
            </toolspecific>
            <place id="place01"/>
        </page>
        <page id="page1">
            <toolspecific tool="nl.rug.ds.bpm.ptnet" version="1.0">
                <process>
                    <groups/>
                    <roles/>
                    <variables/>
                </process>
            </toolspecific>
            <place id="place1"/>
        </page>
        <place id="xx1">
            <initialMarking>
                <text>1</text>
            </initialMarking>
        </place>
        <place id="x1">
            <initialMarking>
                <text>1</text>
            </initialMarking>
        </place>
        <place id="x0">
            <name>
                <text>test_x0</text>
            </name>
        </place>
        <place id="xx0">
            <name>
                <text>test_xx0</text>
            </name>
        </place>
        <toolspecific tool="nl.rug.ds.bpm.ptnet" version="1.0">
            <process>
                <groups/>
                <roles/>
                <variables>
                    <variable name="i" type="var" value="'1'"/>
                    <variable name="x" type="var" value="'0'"/>
                </variables>
            </process>
        </toolspecific>
        <transition id="yy0">
            <name>
                <text>ytest</text>
            </name>
            <toolspecific tool="nl.rug.ds.bpm.ptnet" version="1.0">
                <task isTau="false">
                    <guard>i&gt;=0</guard>
                    <script type="js">i++;</script>
                </task>
            </toolspecific>
        </transition>
        <transition id="y0">
            <name>
                <text>ytest</text>
            </name>
            <toolspecific tool="nl.rug.ds.bpm.ptnet" version="1.0">
                <task isTau="false">
                    <guard>i&gt;=0</guard>
                    <script type="js">i++;</script>
                </task>
            </toolspecific>
        </transition>
    </net>
</pnml>
